conseil-common/src/test/scala/tech/cryptonomic/conseil/common/config/PlatformsTest.scala

```diff
@@ -0,0 +1,43 @@
+package tech.cryptonomic.conseil.common.config
+
+import org.scalatest.{Matchers, WordSpec}
+import tech.cryptonomic.conseil.common.config.Platforms.{
+  BitcoinConfiguration,
+  PlatformsConfiguration,
+  TezosConfiguration,
+  TezosNodeConfiguration
+}
+import tech.cryptonomic.conseil.common.generic.chain.PlatformDiscoveryTypes
+
+class PlatformsTest extends WordSpec with Matchers {
+
+  private val configTezosNode = TezosNodeConfiguration("host", 0, "protocol")
+  private val configTezos = TezosConfiguration("mainnet", enabled = true, configTezosNode, None)
+  private val configBitcoin = BitcoinConfiguration("testnet", enabled = false)
+  private val config = PlatformsConfiguration(List(configTezos, configBitcoin))
+
+  private val platformTezos = PlatformDiscoveryTypes.Platform("tezos", "Tezos")
+  private val platformBitcoin = PlatformDiscoveryTypes.Platform("bitcoin", "Bitcoin")
+
+  private val networkTezos = PlatformDiscoveryTypes.Network("mainnet", "Mainnet", "tezos", "mainnet")
+  private val networkBitcoin = PlatformDiscoveryTypes.Network("testnet", "Testnet", "bitcoin", "testnet")
+
+  "Platforms.PlatformsConfiguration" should {
+      "return enabled platforms, by default" in {
+        config.getPlatforms() should contain only platformTezos
+      }
+
+      "return disabled platforms, when enabled = false" in {
```

Comment:
I'd suggest removing the technical detail of how to search disabled. The description will get out-of-date as soon as we change the code, and probably will not be updated with a refactor.

I'd strive to focus on the what and not the how.